torchvision/io/_video_opt.py

```diff
@@ -26,7 +26,9 @@
         torch.ops.load_library(ext_specs.origin)
         _HAS_VIDEO_OPT = True
 except (ImportError, OSError):
-    pass
+    warnings.warn("Could not import video_reader bakcend."
```

Comment:
```suggestion
    warnings.warn("Could not import video_reader backend."
```

But in general, I would rather avoid having an unconditional warning being raised here. Up to now, this has been an optional backend so it shouldn't warn everytime the user imports torchvision.

I would prefer if we remove this.

Reply:
Alright - I'll revert that commit.